### PR TITLE
add form validation with javascript

### PR DESCRIPTION
**Pull request by PNdunguMaina**
In this project, I did the following:

- added validation for the email input to be in lower characters using javascript

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -252,6 +252,8 @@ <h5>CONTACT US</h5>
               name="user_email"
               placeholder="Email Address"
             />
+            <span class="error"></span>
+
```
Comment (chrystalme) [inline index.html]: Kindly move your error message above the submit button. Also, your error message is supposed to come up after validation, not always as seen in the below image

image

File: main.js [hunk 2]
```diff
@@ -10,3 +10,30 @@ document.querySelectorAll(".nav-link").forEach((n) =>
...
+
+email.addEventListener("input", () => {
+  if (email.value.search(/[a-z]g/) !== -1) {
+    emailError.textContent = "success";
+    emailError.className = "error";
+  } else {
+    showError();
+  }
+});
+
```
Comment (chrystalme) [inline main.js]: This block of code is unnecessary. You want to validate your input field after submit action. Kindly check this [tutorial](https://www.w3schools.com/js/js_validation.asp) to validate a form input

**Review by chrystalme — CHANGES_REQUESTED** [review]
## Hi @PNdunguMaina and @elhamatokhi,

You have done a good job so far!
However, there are some issues that you still need to work on to prepare your project for further evaluation

## Suggested changes ♻️ 
_Check the comments under the review._
Also, 💭 
- Kindly avoid committing folders such as `.vscode` to your repository. 
- Kindly move the following files to your `root` folder out of `html & css files`
             - `.gitignore`
             - `.hintrc`
             - `.stylelintrc.json`
             - `LICENSE`
             - `README.md`
- Check the proper REAME.md [template](https://github.com/microverseinc/readme-template/blob/master/README.md) and adhere to the standard of a professional README
       



## Optional Changes
You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA). Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi, 
Good job so far! @PNdunguMaina 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed, however use a descriptive feature branch:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 

 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request closed without merging**